daemon/cmd/fqdn.go

```diff
@@ -263,20 +263,20 @@ func (d *Daemon) bootstrapFQDN(restoredEndpoints *endpointRestoreState, preCache
 	// restoring after a long delay).
 	globalCache := d.dnsNameManager.GetDNSCache()
 	now := time.Now()
-	for _, restoredEP := range restoredEndpoints.restored {
+	for _, possibleEP := range possibleEndpoints {
```

Comment:
Making `possibleEndpoints` a map (I assume due to the return type of `ReadEPsFromDirNames`?) will lead to random iteration order (see https://blog.golang.org/maps, section _Iteration order_). Could this become a problem?

Reply:
Good point, but that should not make any difference here.